config/turkey.exs

```diff
@@ -0,0 +1,169 @@
+# containing the digested version of static files. This
+# manifest is generated by the mix phx.digest task
+# which you typically run after static files are built.
+config :ret, RetWeb.Endpoint,
+  https: [
+    port: 4000,
+    otp_app: :ret
+  ],
+  http: [
+    port: 4001,
+    otp_app: :ret
+  ],
+  url: [scheme: "https", host: "", port: 443],
+  secondary_url: [scheme: "https", host: "", port: 443],
+  static_url: [scheme: "https", host: "", port: 443],
+  cors_proxy_url: [scheme: "https", host: "", port: 443],
+  assets_url: [scheme: "https", host: "", port: 443],
+  link_url: [scheme: "https", host: "", port: 443],
+  imgproxy_url: [scheme: "http", host: "", port: 5000],
+  pubsub: [name: Ret.PubSub, adapter: Phoenix.PubSub.PG2, pool_size: 4],
+  server: true,
+  root: "."
+
+# Do not print debug messages in production
+config :logger, level: :info
+
+config :ret, Ret.Repo,
+  username: "postgres",
+  password: "postgres",
+  database: "ret_production",
+  hostname: "localhost",
+  template: "template0",
+  pool_size: 10
+
+config :ret, Ret.SessionLockRepo,
+  username: "postgres",
+  password: "postgres",
+  database: "ret_production",
+  hostname: "localhost",
+  template: "template0",
+  pool_size: 10
+
+# ## SSL Support
+#
+# To get SSL working, you will need to add the `https` key
+# to the previous section and set your `:url` port to 443:
+#
+#     config :ret, RetWeb.Endpoint,
+#       ...
+#       url: [host: "example.com", port: 443],
+#       https: [:inet6,
+#               port: 443,
+#               keyfile: System.get_env("SOME_APP_SSL_KEY_PATH"),
+#               certfile: System.get_env("SOME_APP_SSL_CERT_PATH")]
+#
+# Where those two env variables return an absolute path to
+# the key and cert in disk or a relative path inside priv,
+# for example "priv/ssl/server.key".
+#
+# We also recommend setting `force_ssl`, ensuring no data is
+# ever sent via http, always redirecting to https:
+#
+#     config :ret, RetWeb.Endpoint,
+#       force_ssl: [hsts: true]
+#
+# Check `Plug.SSL` for all available options in `force_ssl`.
+
+# ## Using releases
+#
+# If you are doing OTP releases, you need to instruct Phoenix
+# to start the server for all endpoints:
+#
+#     config :phoenix, :serve_endpoints, true
+#
+# Alternatively, you can configure exactly which server to
+# start per endpoint:
+#
+#     config :ret, RetWeb.Endpoint, server: true
+#
+
+# Finally import the config/prod.secret.exs
+# which should be versioned separately.
+import_config "prod.secret.exs"
+
+# Filter out media search API params
+config :phoenix, :filter_parameters, ["q", "filter", "cursor"]
+
+# Disable prepared queries bc of pgbouncer
+config :ret, Ret.Repo, adapter: Ecto.Adapters.Postgres, prepare: :unnamed
+
+config :peerage, log_results: true, via: Peerage.Via.Dns,
+  dns_name: "foo.bar",
+  app_name: "foo"
+
+config :ret, page_auth: [username: "", password: "", realm: "Reticulum"]
+
+config :ret, Ret.Scheduler,
```

Comment:
I wonder if we should DRY this up and import it, seems like a potential footgun to keep multiple copies of it in syc. If we had turkey-only jobs we wanted to run we could do them here, likewise for prod.ess